### PR TITLE
added python example to readme #135

### PR DESCRIPTION
**Pull request by artemlazarev**
Added python example to readme #135

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#230](https://github.com/artipie/artipie/issues/230) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/9b89ac8b-26da-41c6-9672-fd8839255089); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) I see that you unassigned this issue; the order is still assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
This pull request [#230](https://github.com/artipie/artipie/issues/230) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/b9551cbe-eea8-424d-b871-f8009d27e4a5); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -467,6 +467,35 @@ To pull the image use the following command:
...
+    type: fs
+    path: /tmp/artipie/data/python-repo
+```
+
+Publish a package(whl or tar.gz):
+  * Install twine utility, if you don't do it already.
+```bash
+$ python3 -m pip install --user --upgrade twine
+```
+  * build the package, as described in python docs
```
[the remark this answers is not in the document]
Comment (artemlazarev) [reply README.md:487]: @g4s8  sure

**Comment by 0crat** [comment]
Job was finished in 29 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @g4s8[/z](https://www.0crat.com/u/g4s8)[/z](https://www.0crat.com/u/g4s8) didn't make CR comments

**Pull request merged**